py/obj.h

```diff
@@ -421,7 +421,9 @@ typedef struct _mp_rom_obj_t { mp_const_obj_t o; } mp_rom_obj_t;
 // param obj_module: mp_obj_module_t instance
 // prarm enabled_define: used as `#if (enabled_define) around entry`
 
+#ifndef NO_QSTR
```

Comment:
Due to this, qstr extraction will see `MP_REGISTER_MODULE` and it's qstr contents.

But does that mean qstrs for module names will always be included, even if that module is disabled?

Reply:
This has been fixed by removing the 3rd arg of `MP_REGISTER_MODULE()`.  This macro is now only included if the surrounding `#if` / `#endif` wrapper (if it exists) is enabled.